spec/controllers/application_controller_spec.rb

```diff
@@ -84,6 +84,28 @@ def set_accept_language(locale)
 
   end
 
+  describe "print" do
+    render_views
+
+    context "when the SiteSetting is enabled" do
+      before do
+        SiteSetting.stubs(:max_prints_per_hour_per_user).returns(10)
+      end
+
+      it "uses the application layout when there's no param" do
+        get :show, topic_id: topic.id, slug: topic.slug
+        expect(response).to render_template(layout: 'application')
```

Comment:
`render_template` is still supported by RSpec but deprecated in Rails 5 though. The recommended way is to write an integration test and just assert against the response's body directly.